Docs - MiddleManager Affinity "strong" definition (#11480)
apache/druid: docs/configuration/index.md

###### Affinity

> Affinity configs can be provided to the _equalDistribution_ and _fillCapacity_ strategies using the "affinityConfig" field. If not provided, the default is to not use affinity at all.

Reviewer: ```suggestion
Use the `affinityConfig` field to pass affinity configuration to the _equalDistribution_ and _fillCapacity_ strategies. If not provided, the default is to not use affinity at all.
```

|Property|Description|Default|
|--------|-----------|-------|
|`affinity`|JSON object mapping a datasource String name to a list of indexing service MiddleManager host:port String values. Druid doesn't perform DNS resolution, so the 'host' value must match what is configured on the MiddleManager and what the MiddleManager announces itself as (examine the Overlord logs to see what your MiddleManager announces itself as).|{}|

> |`strong`|A boolean determining whether tasks for a datasource must be assigned to affinity-mapped MiddleManagers, causing tasks to remain queued until a slot is available.  If `false`, tasks for a datasource may be assigned to other MiddleManagers whenever affinity-mapped MiddleManagers are not available to run queued tasks.|false|

Reviewer: ```suggestion
|`strong`|When `true` tasks for a datasource must be assigned to affinity-mapped MiddleManagers. Tasks remain queued until a slot becomes available.  When `false`, Druid may assign tasks for a datasource to other MiddleManagers when affinity-mapped MiddleManagers are unavailable to run queued tasks.|false|
```

|--------|-----------|-------|
|`druid.worker.version`|Version identifier for the Indexer.|0|
|`druid.worker.capacity`|Maximum number of tasks the Indexer can accept.|Number of available processors - 1|

> |`druid.worker.globalIngestionHeapLimitBytes`|Total amount of heap available for ingestion processing. This is applied by automatically setting the `maxBytesInMemory` property on tasks.|60% of configured JVM heap| 

Reviewer 2: nit: the blank character at the end of line is unnecessary :sweat_smile:
